README.Rmd

```diff
@@ -52,75 +52,76 @@ All the functions in the elicitr package start with the prefix `elic_`. After th
 #### How elicitr works
 
 Just like you create a form to collect estimates in an elicitation process, the core of elicitr is the creation of an object to store the metadata information. This allows to check whether experts have given their answers in the expected way.\
-Any analysis starts by creating this object with the `start` function. Then, data can be added and retrieved using the `add_data` and `get_data` functions respectively. Finally data can be plotted using the `plot` function. Details about the implementation and example usages of these functions can be seen bellow.
+Any analysis starts by creating this object with the `start` function. Then, data can be added and retrieved using the `add_data` and `get_data` functions respectively. Finally data can be plotted using the `plot` function. Details about the implementation and example usages of these functions based on the simulated datasets inluded in elicitr can be seen bellow.\
+Their are five simulated datasets provided in elictr. They are intended to demonstrate the functionality of the package and do not represent an actual elicitation process (names are also randomly generated).
```

Comment:
Line 56: change "Their are five simulated datasets provided in elictr." with "In total, there are five simulated datasets provided with the package."

Reply:
thank you for seing the typo (their/there). I think the correct form is provided "in" the package and not "with" the package.

Reply:
Either "provided with" or "included in", but "provided in".